tests/acceptance/workflow_test.py

```diff
@@ -112,3 +113,71 @@ def test_walk_execution_history(workflow_client, decision_client):
     decision_task = poll_decision_and_respond(decision_client)
     terminate_workflow(workflow_client, workflow_id)
     walk_execution_history(decision_client, workflow_id, decision_task.workflow_run_id)
+
+
+def test_count_open_workflow_executions(workflow_client):
```

Comment:
Hmm I think the boto_client we are using is a mock thus cannot raise exceptions, since we want the validation to happen on boto_client only? The integration test has covered it though if you want to have a look.